tests/unit/data/test_metadata_handler.py

```diff
@@ -0,0 +1,103 @@
+import os
+import pytest
+
+from clean_air.data.metadata_handler import MetadataHandler, dict_printer
+from clean_air.data.data_filter_handler import print_yaml_file
+
+#NB: These tests use the test data :
+# station_metadata.yaml
+# metadata_one.yaml
+# antartic_metedata.yaml
+
+
+#('METADATA TEST', 'Build list of all metadata files and turn filters to on by default \n')
+@pytest.fixture()
+def mh(sampledir):
+    mh = MetadataHandler(os.path.join(sampledir,"test_data"))
+    dict_printer(mh.filters_dict)
+    return mh
+
+@pytest.fixture()
+def metadata_filepath():
+    filepath = os.path.join("net","home","h05","clucas","PycharmProjects","CleanAirProject","clean_air","data")
```

Comment:
I think these hardcoded paths will only work on your local machine, not on github or anyone else's VDI. Looking at the others, I think you just missed this occurrence when you did the others